src/Commenter.ts

```diff
@@ -0,0 +1,52 @@
+import * as github from '@actions/github';
+import * as core from '@actions/core';
+import { GetResponseDataTypeFromEndpointMethod } from '@octokit/types';
+
+import assert from 'assert';
+import fs from 'fs';
+
+export default class Commenter {
+  private readonly githubToken: string;
+
+  constructor(private readonly filePath: string) {
+    this.githubToken = core.getInput('github-token');
```

Comment:
`github-token` is defined here - https://github.com/getappmap/analyze-action/blob/0e8592c6f87c92792fc5250bd539163efcfd4c84/src/preflight.ts#L30 - so pass it in as a parameter. This makes it easier to test as well, and removes a dependency of this on `@actions/core`.

Reply:
Okay, that makes sense, I changed it.